package.json

```diff
@@ -57,6 +57,8 @@
     "dripip": "^0.9.0",
     "express": "^4.17.1",
     "fetch-cookie": "0.7.2",
+    "graphql": "^15.3.0",
```

Comment:
Because we're exposing a type from `graphql` package to users of this package I think we need to turn this into a production dep. But I really don't want to do that because `graphql` is usually used as a peer dep and users want to manage the version themselves.

Reply:
Just tried a PR release, example:

![image](https://user-images.githubusercontent.com/284476/89193983-c2d6c880-d574-11ea-940c-ff7455f351b8.png)

Reply:
made into peer dep